azure-pipelines.yml

```diff
@@ -263,34 +224,12 @@ jobs:
       includePatterns:
         - "/hw/top_earlgrey/Vchip_earlgrey_verilator"
 
-- job: execute_verilated_tests_englishbreakfast
-  displayName: Execute tests on the Verilated English Breakfast toplevel design
-  pool: ci-public
-  dependsOn:
-    - chip_englishbreakfast_verilator
-    - sw_build_englishbreakfast
-  steps:
-  - template: ci/install-package-dependencies.yml
-  - template: ci/download-artifacts-template.yml
-    parameters:
-      downloadPartialBuildBinFrom:
-        - chip_englishbreakfast_verilator
-        - sw_build_englishbreakfast
-  - bash: |
-      # Install an additional pytest dependency for result upload.
-      pip3 install pytest-azurepipelines
-
-      . util/build_consts.sh
-      pytest --version
-      pytest test/systemtest/englishbreakfast/test_sim_verilator.py \
-        -m "not slow" \
-        --log-cli-level=DEBUG \
-        --test-run-title="Run English Breakfast system tests with Verilator simulation" \
-        --napoleon-docstrings
-    displayName: Execute tests
-
-# TODO: change this name and replace sw_build_englishbreakfast and execute_verilated_tests_englishbreakfast
-# once testing shows it's stable
+# Software targeting the English Breakfast top level is produced by patching
+# the source tree before building. This builds a selected subset of software
+# only.
+# TODO: This is a rather ugly hack, which will go away once we properly support
+# building more than one top-level design with different parametrizations.
+# Work towards this goal is tracked in issue #4669.
 - job: bazel_build_and_execute_verilated_tests_englishbreakfast
```

Comment:
Change this job name to `build_and_execute_verilated_tests_englishbreakfast` and make the corresponding change in the dependency names under the `chip_englishbreakfast_cw305` job

Reply:
ah right. thanks. fixed.